xiblint/rules/strict_customization.py

```diff
@@ -0,0 +1,41 @@
+from xiblint.rules import Rule
+
+
+class StrictCustomization(Rule):
+    """
+    Ensures given system types are subclassed by a set of classes.
+
+    You must specify a module as part of the class name.
+
+    Example configuration:
+    {
+      "custom_classes": {
```

Comment:
because this conflicts with https://github.com/lyft/xiblint/pull/35#discussion_r236471041 maybe these should be more specific to these rules?

Reply:
Resolved in the other comment. Doesn't conflict since it's nested under the rule name.